### DISCUSSION TITLE
numcin

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#407] [Tuning parameters]
Here is a list of runs exploring impact numcin.

**FLTHIST**
- control (`numcin = 1`): https://github.com/NCAR/amwg_dev/issues/353
- `numcin = 1->2`: https://github.com/NCAR/amwg_dev/issues/400
- `numcin = 1->3`: https://github.com/NCAR/amwg_dev/issues/401

**FMTHIST**
- control (`numcin = 1`): https://github.com/NCAR/amwg_dev/issues/374
- `numcin = 1->2` + dust retuning: https://github.com/NCAR/amwg_dev/issues/405

**BLT1850**
- control (`numcin = 1`): https://github.com/NCAR/amwg_dev/issues/391
- `numcin = 1->2`: https://github.com/NCAR/amwg_dev/issues/406

@PeterHjortLauritzen, @swrneale, @islasimpson, @JulioTBacmeister, @adamrher, @tilmes

**Comment by swrneale** [comment]
@cecilehannay I still think #405 may have a dust retuning change in addition to numcin=2, could you check?

**Reply by cecilehannay** [reply]
Hi Rich, 
Your run are right that there is a dust retuning. 
@tilmes: could you change the description of your run #405 to reflect the dust retuning. Thanks a bunch.

**Comment by swrneale** [comment]
@justin-richling would you have time to run the diagnostics of #400 vs. #353 and #401 vs. #353? Thanks!
Hopefully we have 1996-2005 for all the runs?

**Reply by cecilehannay** [reply]
@swrneale: I had already run these diags but it was for years 1995-2005. 
https://webext.cgd.ucar.edu/FLTHIST/f.cam6_3_119.FLTHIST_ne30.r328_gamma0.33_soae_numcin2.001/atm/
https://webext.cgd.ucar.edu/FLTHIST/f.cam6_3_119.FLTHIST_ne30.r328_gamma0.33_soae_numcin3.001/atm/

Do you also need 1996-2005?

**Reply by justin-richling** [reply]
I started running the diags for 1995-2005 but with TEM diags. I can let them keep running and just update the website with TEM or I can just kill it now.

**Comment by islasimpson** [comment]
There doesn't seem to be a clear story regarding the influence of numcin on the JJA 200hPa Pacific jet issue.  The issue seems to be less bad in the F-case but increasing numcin looks like it makes it worse.  The issue is worse in the B-case but increasing numcin seems to make it better.  So it seems like there's an opposite influence of numcin depending on whether you're in a B-case or an F-case.   I will look into it more to see how much we should actually care about this particular level later.

<img width="883" alt="Screen Shot 2023-09-26 at 12 36 48 PM" src="https://github.com/NCAR/amwg_dev/assets/41842751/9e8662e8-cea8-4039-9ce5-f55c68616955">

**Reply by swrneale** [reply]
@isla could you do this lat-lon slice at something like 50mb as Simone showed a horrible increase in the stratospheric jet with numcin=2 in her FMTHIST run #405.

**Reply by islasimpson** [reply]
I'm not really seeing that.  The stratospheric jet in the SH is definitely stronger in F-cases than in B-cases during SON.  Could that be the reason?  Below are zonal mean zonal winds for four seasons.  I don't see much dependence on numcin.

<img width="760" alt="Screen Shot 2023-09-26 at 3 24 22 PM" src="https://github.com/NCAR/amwg_dev/assets/41842751/1840fe71-5797-4b14-86e4-47bd6a8c5cf8">

<img width="761" alt="Screen Shot 2023-09-26 at 3 25 33 PM" src="https://github.com/NCAR/amwg_dev/assets/41842751/588f8aea-a139-469e-9ba7-c7095cee1394">

<img width="759" alt="Screen Shot 2023-09-26 at 3 26 36 PM" src="https://github.com/NCAR/amwg_dev/assets/41842751/ed1d944a-34a8-4a73-8b9d-c31be3e27915">

<img width="760" alt="Screen Shot 2023-09-26 at 3 27 44 PM" src="https://github.com/NCAR/amwg_dev/assets/41842751/532be2d1-e5d0-4dfc-ae1b-26b05c51f58d">

**Reply by tilmes** [reply]
@islasimpson are you comparing the MT model https://github.com/NCAR/amwg_dev/issues/405 ?

Why does it cut off at the top?

**Reply by swrneale** [reply]
Hmm. @tilmes could you post that figure of the strong JJA strat. jet with numcin=2 you showed at the WACCM meeting?

**Reply by islasimpson** [reply]
I am not comparing the mid-top.  These are all the low top simulations.  That's why it cuts off at the top.  I can plot the mid-top now.

**Reply by tilmes** [reply]
I checked the original diagnostics, and grabbed JJA, instead of SON, here is the correct comparison. The jet is not as massively increased, but still some:
<img width="491" alt="image" src="https://github.com/NCAR/amwg_dev/assets/22222754/ec48b161-058e-47e0-9655-2a6e1f362a1d">

**Reply by tilmes** [reply]
here are the diagnostics compared to a nudged MERRA2 run:
https://acomstaff.acom.ucar.edu/tilmes/amwg/cam7/f.cam6_3_119.FMTHIST_ne30.r328_gamma0.33_soae.numcin2.001.1997_2000-f.cam6_3_119.FMTHIST_ne30.r328_gamma0.33_soae.nudged.001.2010_2012/wset4/wset4.htm

**Reply by islasimpson** [reply]
Here's the mid-top SON comparison.  The numcin1 run in the comparison below is 12 years and the numcin2 run is 6 years, which is still short.  I don't think it's a good idea to look at comparisons of zonal mean zonal wind between 2 or 3 year runs like above in SON which is the most variable time of the year in the SH due to interannual variations in the vortex breakdown date.

<img width="329" alt="Screen Shot 2023-09-26 at 3 51 43 PM" src="https://github.com/NCAR/amwg_dev/assets/41842751/005ab36c-c539-4ef1-b3be-3393de03c7d9">

**Reply by swrneale** [reply]
Phew, well this looks more reasonable at least. 
Looking at our diagnostics, I am also starting to wonder whether beginning our AMIP simulations in 1996 is a good idea since the huge 1997/98 ENSO will give a skewed view of the real climatology that obs. show. Thoughts?
 
@cecilehannay @PeterHjortLauritzen @JulioTBacmeister @islasimpson @adamrher

**Reply by tilmes** [reply]
@swrneale it is still a large increase in the polar vortex, just not as visible in Isla's plots... but I agree, we have to see now much the strong vortex in the first place is tunable.. sorry for showing preliminary results at the WACCM meeting.

**Comment by JulioTBacmeister** [comment]
I think Isla's point about SON winds in the Southern Hemisphere is a good
one. This is a very variable aspect of the climo by SH standards. See image below.

**Comment by JulioTBacmeister** [comment]
This is an image showing a 20-year zonal average U for SON in coupled runs 42(NUMCIN=1) and 48 (NUMCIN=2). In BLT it seems NUMCIN has little impact on ths feature

![image](https://github.com/NCAR/amwg_dev/assets/9221710/f000ed60-c734-46cf-89df-7f573faabd0d)
